package.json

```diff
@@ -109,7 +109,8 @@
     "grommet-theme-aruba": "^0.1.1",
     "grommet-theme-dxc": "^0.1.1",
     "grommet-theme-hp": "^0.1.1",
-    "grommet-theme-hpe": "^0.4.0",
+    "grommet-theme-hpe-new": "https://github.com/grommet/grommet-theme-hpe/tarball/stable",
```

Comment:
A minor point for consistency, what about naming the old one "grommet-theme-hpe-old" instead? Or, perhaps across the board to use "hpe-v0"?

Reply:
Done and Done.